.github/workflows/ci.yml

```diff
@@ -139,6 +139,45 @@ jobs:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
 
+  docker:
+    needs:
+      - lint
+      - test-unit
+      - test-integration
+      - validate-components
+    # Only run on tags for now
+    if: ${{ github.ref_type == 'tag' }}
```

Comment:
FYI `docs` runs on `if: startsWith(github.ref, 'refs/tags/v')`; but we don't have many other tags in any case.

Reply:
> > ```yaml
> >       - docs
> > ```
> 
> I don't see that one though; not a major issue in any case (and perhaps even better).

Yes, intentionally left it out, but my description still mentioned it, (typo)

Reply:
`if: startsWith(github.ref, 'refs/tags/v'` is better, will change it

Reply:
Same with me; I just wondered what we want.

Reply:
I think it is better, because if someone accidentally tags a version otherwise (or has some valid reason to do so), we don't necessarily want it to end up on docker hub.